sootty/save.py

```diff
@@ -0,0 +1,66 @@
+import os
+import subprocess
+
+
+def save_query(save, name, wires, br, length, start, end, display):
+    save = os.getenv("HOME") + "/.config/sootty/save/" + save
```

Comment:
It would be nice if all of the saved queries could be stored in one file rather than having a separate one for each query. (You can use something like yaml or json for storing a dict of queries)

Reply:
Yeah, I was thinking about adding that as an update to this feature as well. Will look into this.